src/saving/SaveHelper.cs

```diff
@@ -211,6 +206,42 @@ public static void DeleteSave(string saveName)
         }
     }
 
+    public static void RemoveExcessAutoAndQuickSaves()
```

Comment:
This is not what I had in mind, this causes problems:
- if the user has autosave disabled, but the number of autosaves turned down
- And they make a quick save, that then deletes their autosaves they probably didn't want deleted.

Instead you should make a general method like
`DeleteExcessSaves(string nameStartsWith, int maximumCount)` that is then used by both the auto save and quick save deletion.

Reply:
Ah I see. Thanks!